### PR TITLE
Set up GitHub Actions and fix wrapper script

### PR DESCRIPTION
**Pull request by Goooler**
Validate builds for PRs and pushes on master.

Check the result at https://github.com/Goooler/Notally/actions/runs/9398823467/job/25884993374.

### CODE REVIEW

File: gradlew
Comment (Goooler) [inline gradlew:1]: Update this due to https://github.com/Goooler/Notally/actions/runs/9398672560/job/25884505156

**Comment by OmGodse** [comment]
What exactly is the point of this?

**Comment by Goooler** [comment]
https://github.com/features/actions

**Comment by clach04** [comment]
@Goooler I can see from your last Action (from June) https://github.com/Goooler/Notally/actions/runs/9398823467/job/25884993374 that this does a sanity check build (potentially useful for sanity checking PRs). Does this make test/installable APKs available?

**Comment by Goooler** [comment]
That makes sure the assemble tasks pass, we can upload the build artifacts as well.

**Pull request closed without merging**